components/description-list/demo/description-list-test.js

```diff
@@ -0,0 +1,136 @@
+import '../../icons/icon.js';
```

Comment:
Is this still used?  Missing a test you wanted to include?

Reply:
good eye! Not used anymore since I replaced that demo